.circleci/real_config.yml

```diff
@@ -12,9 +12,6 @@ orbs:
   win: circleci/windows@5.0.0
 
 executors:
-  python-37:
-    docker:
-      - image: python:3.7-slim-buster
   python-38:
     docker:
       - image: python:3.8-slim-buster
```

Comment:
We're still using the Buster images? :D 

They've moved through Bullseye and are now on Bookworm.  This is probably a separate PR, but those buster images haven't been updated for about a year.  docker-library/python#822

Reply:
https://github.com/determined-ai/determined/pull/8943